src/templater.js

```diff
@@ -51,7 +51,7 @@
     }
 
     // return a function to use the replacement values
-    return function(data) {
+    return function(data, should_escape = true) {
```

Comment:
```suggestion
    return function(data, opts) {
```

I'd prefer to see this as an object in case there's other configuration in the future